extension/src/config/config.js

```diff
@@ -19,6 +38,14 @@ function getCtpConfig(ctpProjectKey) {
         ctpProjectKey
       )}]`
     )
+  if (!_isValidAuthenticationConfig(ctpConfig)) {
```

Comment:
I think this check should be in `loadAndValidateConfig` to fail fast. If it's like this, I could imagine that server could start but it would not know that one of the authConfig is wrong until it's used.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/584/commits/75fbc5fa5ce753191778a1cdbf6ddabf334686c7